packages/dds-components/src/components/ToggleButton/ToggleButton.module.css

```diff
@@ -50,6 +49,16 @@
   }
 }
 
+.small {
+  gap: var(--dds-spacing-x0-25);
+  padding: var(--dds-spacing-x0-5) var(--dds-spacing-x1);
+}
+
+.xsmall {
+  gap: var(--dds-spacing-x0-125);
+  padding: var(--dds-spacing-x0-25) var(--dds-spacing-x1);
+}
+
 .content--with-icon {
   padding-inline-start: var(--dds-spacing-x0-75);
 }
```

Comment:
`.content--with-icon` må nå ha variant for hver størrelse; Det er `var(--dds-spacing-x0-25)` i `xsmall`.